remoting/src/main/java/org/apache/rocketmq/remoting/protocol/RemotingCommand.java

```diff
@@ -64,7 +65,7 @@ public class RemotingCommand {
     private static final String BOOLEAN_CANONICAL_NAME_1 = Boolean.class.getCanonicalName();
     private static final String BOOLEAN_CANONICAL_NAME_2 = boolean.class.getCanonicalName();
     private static volatile int configVersion = -1;
-    private static AtomicInteger requestId = new AtomicInteger(0);
+    private static final AtomicInteger requestId = new AtomicInteger(0);
```

Comment:
Please move this unrelated modification to another PR.

Reply:
done